src/integrationTest/java/uk/gov/hmcts/reform/sendletter/controllers/GetLetterStatusTest.java

```diff
@@ -48,35 +46,43 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private LetterRepository letterRepository;
+
     @MockBean
     private AuthTokenValidator tokenValidator;
 
-    @SpyBean
-    private LetterRepository letterRepository;
+    @After
+    public void tearDown() {
+        letterRepository.deleteAll();
+    }
 
     @Test
     public void should_return_200_after_creating_single_letter_in_db() throws Exception {
```

Comment:
doesn't match what's tested inside (was wrong before, but maybe you could fix it as part of this PR?)